### PR TITLE
Add fetch step for wv/dhhr. Fixes #368.

### LINKED ISSUE #368: fetch wv/dhhr
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://dhhr.wv.gov/News/2021/Pages/COVID-19-Vaccination-Clinics-March-2-7,-2021.aspx

Put your script in a file named: `wv/dhhr/fetch.sh` (or `fetch.sh`, etc.)

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch wv/dhhr
    ```

### PR DESCRIPTION
**Pull request by skbdpup**
# Fetch dhhr for WV

| Key Details |
|-|
| Resolves #368 |
State: WV |
Site: dhhr |

## Data sample
```html
<p style="margin:0in"><b><u><span style="font-size:11pt"><font color="#000000" face="Calibri">Tuesday, March 2, 2021</font></span></u></b></p>
<font color="#000000" face="Times New Roman" size="3">

</font><p style="margin:0in;text-align:justify"><b><span style="font-size:11pt"><font color="#000000" face="Calibri">Cabell County (All clinic appointments have
been assigned)</font></span></b></p>
<font color="#000000" face="Times New Roman" size="3">

</font><p style="margin:0in;text-align:justify"><b><span style="color:black;font-family:calibri, sans-serif;font-size:11pt">11:00 a.m. - 4:00 p.m.</span></b><span style="font-family:calibri, sans-serif;font-size:11pt"><font color="#000000">,<b> </b></font></span><span style="color:black;font-family:calibri, sans-serif;font-size:11pt">COVID-19 Vaccine Center (old Sears store)</span><span style="font-family:calibri, sans-serif;font-size:11pt"><font color="#000000">, </font></span><span style="color:black;font-family:calibri, sans-serif;font-size:11pt">100
Huntington Mall Road, Barboursville, WV 25504</span><span style="font-family:calibri, sans-serif;font-size:11pt"><font color="#000000">. By appointment only.</font></span></p>
```

## Before Opening a PR
- [x] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [x] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

File: vaccine_feed_ingest/runners/wv/dhhr/fetch.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+output_dir=""
+if [ -n "${1}" ]; then
+    output_dir="${1}"
+else
+    echo "Must pass an output_dir as first argument"
```
Comment (skbdpup) [inline vaccine_feed_ingest/runners/wv/dhhr/fetch.sh:9]: Should we `exit 1` here?
And should we print the error message to stderr?

**Review by bryanculbertson — CHANGES_REQUESTED** [review]
Thank you for working on this fetch stage! It looks like this site is going to be a little harder than most since they use a different URL for each weekly update of the clinics!

Can you make this fetcher smarter? It needs to grab the latest "Vaccination Clinics" URL from the news page: https://dhhr.wv.gov/News/2021/Pages/default.aspx

Which for this week is: `https://dhhr.wv.gov/News/2021/Pages/COVID-19-Vaccination-Clinics-May-3-8,-2021.aspx`

And then fetch that url.

Thank you!

**Comment by bryanculbertson** [comment]
@skbdpup I just noticed that you requested a review 3 days ago! Sorry about missing this, reviewing now.

**Review by bryanculbertson — APPROVED** [review]
This looks great. Thank you for figuring out how to grab the latest URL, and for documenting the process.

**Pull request merged**